app/models/hubstats/label.rb

```diff
@@ -18,10 +18,10 @@ class Label < ActiveRecord::Base
     has_and_belongs_to_many :pull_requests, :join_table => 'hubstats_labels_pull_requests'
 
     def self.first_or_create(label)
-      if exists = Hubstats::Label.where(name: label[:name]).first
```

Comment:
I was thinking about consistently with remaining code, but after looking at more files, I think it might be easier to leave some with the `Hubstats::` since there are so many instances of it.